lib/alert/addon/components/alert/time-duration/component.js

```diff
@@ -20,4 +20,13 @@ export default Component.extend({
 
     set(this, 'value', h * 3600 + m * 60 + s);
   }.observes('h', 'm', 's'),
+
+  setTime: function() {
+    const value = get(this, 'value') || 0;
+    const d = moment.duration(value * 1000);
+
+    set(this, 'h', d.hours());
```

Comment:
The following three sets should use this helper method `setProperties(this, { h: ..., ...})`